src/AlohaKit.Gallery/AlohaKit.Gallery.csproj

```diff
@@ -64,9 +57,18 @@
 	  <ProjectReference Include="..\AlohaKit\AlohaKit.csproj" />
 	</ItemGroup>
 
+	<ItemGroup>
+	  <PackageReference Include="Microsoft.Maui.Controls" Version="8.0.61" />
+	  <PackageReference Include="Microsoft.Maui.Controls.Compatibility" Version="8.0.61" />
```

Comment:
Is the Compatibility package actually required?

Reply:
I think no. Removed it